<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramKeyGripV2</class>
 <widget class="QMainWindow" name="ProgramKeyGripV2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radioButton_one">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
color: black;
}

QRadioButton::indicator {
width: 11px;
height: 11px;
border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
border: 2px solid; 
border-color: green;
border-radius: 7px;
background-color: white; 
width: 10px; 
height: 10px; 
}

QRadioButton::indicator::checked{ 
border: 2px solid; 
border-color: green;
border-radius: 7px;
background-color: black; 
width: 10px; 
height: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Segment 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_two">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
color: black;
}

QRadioButton::indicator {
width: 11px;
height: 11px;
border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
border: 2px solid; 
border-color: red;
border-radius: 7px;
background-color: white; 
width: 10px; 
height: 10px; 
}

QRadioButton::indicator::checked{ 
border: 2px solid; 
border-color: red;
border-radius: 7px;
background-color: black; 
width: 10px; 
height: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Segment 2 </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_three">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
color: black;
}

QRadioButton::indicator {
width: 11px;
height: 11px;
border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
border: 2px solid; 
border-color: purple;
border-radius: 7px;
background-color: white; 
width: 10px; 
height: 10px; 
}

QRadioButton::indicator::checked{ 
border: 2px solid; 
border-color: purple;
border-radius: 7px;
background-color: black; 
width: 10px; 
height: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Segment 3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_four">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {

color: black;
}

QRadioButton::indicator {
width: 11px;
height: 11px;
border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
border: 2px solid; 
border-color: blue;
border-radius: 7px;
background-color: white; 
width: 10px; 
height: 10px; 
}

QRadioButton::indicator::checked{ 
border: 2px solid; 
border-color: blue;
border-radius: 7px;
background-color: black; 
width: 10px; 
height: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Segment 4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_five">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {

color: black;
}

QRadioButton::indicator {
width: 11px;
height: 11px;
border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
border: 2px solid; 
border-color: green;
border-radius: 7px;
background-color: white; 
width: 10px; 
height: 10px; 
}

QRadioButton::indicator::checked{ 
border: 2px solid; 
border-color: green;
border-radius: 7px;
background-color: black; 
width: 10px; 
height: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Segment 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_one">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_two">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_three">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_four">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_five">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back_keypro">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa,
                stop :   0.5 #ffbb6e, stop :   0.55 #feae42, stop :   1.0 #fedb74);
}

QPushButton {
        border: 1px solid #6593cf;
        border-radius: 2px;
        padding: 5px 15px 2px 5px;
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #f5f9ff,
                stop :   0.5 #c7dfff, stop :   0.55 #afd2ff, stop :   1.0 #c0dbff);
        color: #0000;
        font:  large &quot;Arial&quot;;
		font-size: 10px;
        height: 30px;
}




QPushButton:pressed {
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #c0dbff,
        stop :   0.5 #cfd26f, stop :   0.55 #c7df6f, stop :   1.0 #f5f9ff);
        padding-top: 2px;
        padding-left: 3px;

}
QPushButton:on {
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #5AA72D,
        stop :   0.5 #B3E296, stop :   0.55 #B3E296, stop :   1.0 #f5f9ff);
        padding-top: 2px;
        padding-left: 3px;
}

QPushButton:disabled {
        background: transparent #e5e9ee;
        padding-top: 2px;        
        padding-left: 3px;
        color: black;
}</string>
    </property>
    <property name="text">
     <string> Back to Program</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>49</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
